test/functional/editor_controller_test.rb

```diff
@@ -45,8 +45,10 @@ def setup
         tags: 'question:question,one',
         template: 'question',
         redirect: 'question'
-    assert_select "h3", "Ask a question of the community"
-    assert_select "input#taginput[value=?]", "question:question,one"
+        assert_response :redirect
```

Comment:
Yes, unless you want to try modifying them to fit your new template -- and in fact, both may still work, since your template is almost identical to the previous one. Each looks for an HTML element (`h3` or `input`) and the second also looks to see that the `<input>` it finds has the specified value. They will actually reference these two lines in your new template: 

* https://github.com/publiclab/plots2/pull/944/files#diff-4be9cc3eba98e96c36dddfafeae8a472R78
* https://github.com/publiclab/plots2/pull/944/files#diff-4be9cc3eba98e96c36dddfafeae8a472R95

So, try uncommenting them to see if they pass!